### DISCUSSION TITLE
Add ability to hide pinned messages + slim banner

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#16647] [General]
It would be awesome to have the ability to hide pinned messages. There's some things (like contact info) that I always want pinned for conveinence, but don't want to constantly see on the screen.

It would also be great if the banner could be thinner. On Signal and Telegram they are half the height.

![Screenshot_2022-12-22-21-49-48-98_435383e61849826ed84c38f611283930](https://user-images.githubusercontent.com/65834069/209267412-f3ad3cec-0a5a-40a9-bf1b-31f791a72a58.jpg)

Signal banners for reference:

![Screenshot_2022-12-22-21-53-20-31_4b14b257287ac0a6d1fd5fbb2487716f](https://user-images.githubusercontent.com/65834069/209267764-d33e5e69-2d26-4438-a81d-a48c98569d93.jpg)

**Comment by flexsurfer** [comment]
cc @pedro-et @xAlisher
